slides/04-networking-in-riot/networking-in-riot.md

```diff
@@ -291,7 +305,7 @@ void sock_<type>_close(sock_<type>_t *sock);
 
 ## Exercise: UDP client-server application
 
-Follow the instructions in the notebook **riot/networking/sock-udp/sock-udp.ipynb**
+Follow the instructions in the notebook **riot/networking/udp-sock/udp-sock.ipynb**
```

Comment:
You also have to change the line in the input tag otherwise the automatic opening of notebook still doesn't work